src/package_io/parse_input.star

```diff
@@ -92,7 +91,10 @@ def parse_input(plan, input_args):
             )
         )
 
-    result["tx_spammer_params"] = get_default_tx_spammer_params()
+    result["tx_spammer_params"] = dict(
```

Comment:
Nice find; to keep it consistent can we follow the patter we do with the `mev_params`; see line 63. If you specify this in that area then the for loop below it will mix the defaults and what was siet

Reply:
@h4ck3rk3y Applied :) Tested and works well